data/en/ukis_0001.json

```diff
@@ -0,0 +1,4561 @@
+                                "min_value": {
+                                    "value": 0,
+                                    "exclusive": false
+                                },
+                                "decimal_places": 2,
+                                "currency": "GBP"
+                            }]
+                        }]
+                    },
+                    {
+                        "id": "block2440",
+                        "type": "Question",
+                        "questions": [{
+                            "id": "question2440",
+                            "title": "Did {{ metadata.trad_as_or_ru_name }} invest in the <em>acquisition of existing knowledge</em> for the purposes of current or future innovation?",
+                            "guidance": {
+                                "content": [{
+                                        "description": "<strong>Include: </strong>"
+                                    },
+                                    {
+                                        "list": [
+                                            "purchase or licensing of patents and non-patented inventions, know-how and other types of knowledge from other businesses or organisations"
+                                        ]
+                                    }
+                                ]
+                            },
+                            "description": "",
+                            "type": "General",
+                            "answers": [{
+                                "id": "answer3182",
+                                "mandatory": false,
+                                "type": "Radio",
+                                "label": "",
+                                "description": "",
+                                "options": [{
+                                        "label": "Yes",
+                                        "value": "Yes"
+                                    },
+                                    {
+                                        "label": "No",
+                                        "value": "No"
+                                    }
+                                ]
+                            }]
+                        }],
+                        "routing_rules": [{
+                                "goto": {
+                                    "block": "block2441",
+                                    "when": [{
+                                            "id": "answer3182",
+                                            "condition": "not equals",
+                                            "value": "No"
+                                        },
+                                        {
+                                            "id": "answer3182",
+                                            "condition": "set"
```

Comment:
these `set` conditions seem redundant? as it cant be `No` if its not set?

Reply:
Done by author.

I'll update all the author rules to simplify it.

Reply:
Are these something publisher adds in automatically, or is the person defining the survey adding them?

Reply:
I believe its the way publisher does it because of some reason I don't know.